docs/ExpoUpdates.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Expo Updates
+description: Setting up Ignite to deployed over-the-air (OTA) updates
+tags:
+  - Expo
+  - expo-updates
+  - EAS Update
+  - Expo Updates
+last_update:
+  author: Frank Calise
+---
+
+# Expo Updates
+
+This guide will teach you how to set up over-the-air (OTA) updates with Expo and EAS Update within an Ignite project.
+
+## Appetizer
+
+Follow the [Pristine Expo Project](./PristineExpoProject.md) recipe first to make sure you're starting with an Expo only project.
+
+You'll also need `eas-cli` globally installed and and an [Expo account](https://expo.dev/signup) if you don't already have one.
+
+`npm install -g eas-cli`
+
+## Steps
+
+### 1. Project Setup
+
+From within your project directory, run the following:
+
+```bash
+npx expo install expo-updates
+eas build:configure
+```
+
+- Answer yes to setting up the EAS project prompt and you can configure it for both platforms (**note:** for the purposes of this guide we'll be using Android)
+- This will make some edits to your `app.json` now that the project has an EAS identifier, as well as add a new `eas.json` configuration file.
+
+```bash
+eas update:configure
+```
+
+- Some more `app.json` changes will be made adding an `updates` key to your configuration file.
+
+### 2. Edit Build Profiles
+
+Modify the newly generated `eas.json` to configure a `preview` build profile. We'll be using a `buildType` of `apk` for the Android build to keep things easy for testing, your production build can still use an Android App Bundle.
+
+```json
+{
+  "cli": {
+    "version": ">= 3.0.0"
+  },
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    },
+    "preview": {
+      "channel": "preview",
+      "android": { "buildType": "apk" },
+      "ios": { "simulator": false }
+    },
+    "production": {}
+  },
+  "submit": {
+    "production": {}
+  }
+}
+```
+
+### 3. Create the Build
+
+Next, we'll create a build that we can test with on either an Android emulator or device. You may do this via EAS or [locally](https://docs.expo.dev/build-reference/local-builds/) if you
+
+```bash
+# via EAS
+eas build --profile preview -p android
+
+# locally
+eas build --profile preview -p android --local
+```
+
+Accept the prompts for generating the new Android Keystore. Once that is completed, drop it on your emulator or device and fire it up! 🔥
```

Comment:
Might just want to let them know that the above steps can take a while 😂 . Took me about 7 minutes total haha